inc/class-wpseo-replace-vars.php

```diff
@@ -479,7 +479,7 @@ private function retrieve_term_description() {
 		if ( isset( $this->args->term_id ) && ! empty( $this->args->taxonomy ) ) {
 			$term_desc = get_term_field( 'description', $this->args->term_id, $this->args->taxonomy );
 			if ( $term_desc !== '' ) {
-				$replacement = trim( strip_tags( $term_desc ) );
+				$replacement = trim( wp_strip_all_tags( $term_desc ) );
```

Comment:
As `wp_strip_all_tags` already trims the content, this does not have to be wrapped again.